### PR TITLE
Add 2021 report

### PR DESCRIPTION
**Pull request by meatballs**

### CODE REVIEW

**Comment by doismellburning** [comment]
Nicely done, looks great to me, thank you

**Review by AnnJBarr — APPROVED** [review]
Looks good to me. Do we as trustees approve it by meeting in advance of the AGM or is our approval on here deemed acceptance?
Also any timescales for closure of PUKSL?

**Comment by meatballs** [comment]
> Do we as trustees approve it by meeting in advance of the AGM or is our approval on here deemed acceptance?

Approval on here is fine.

**Comment by drvinceknight** [comment]
Looks good to me. Thank you @meatballs

**Review by cecilialiao — APPROVED** [review]
2021 report looks good. Thanks!

**Comment by Gailogail** [comment]
Looks good to me too. Just that same question about PUKSL, as it was an intention in last year's report and there's no update on that.

**Review by Gailogail — APPROVED** [review]

File: source/2021.tex [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            If there were a reduction in membership of the UKPA then there would have to be a contraction, consolidation or closure of its activities.
+
+            \subsubsection{Injury to Members, Conference Attendees or Trustees}
+            The UKPA requires insurance to cover this potential risk and the trustees intend to put that insurance in place once the UKPA has sufficient funds to cover the cost of the premium.
+
+            \subsubsection{Reduction or Loss of Volunteers}
+            The UKPA is totally reliant upon volunteers to run and administer the activities of the UKPA. If there were a reduction in the number of volunteers to an unacceptable level, then there would have to be a contraction, consolidation or closure of activities.
+
+    \section{Achievements and Performance}
+    	As a result of the ongoing COVID19 pandemic, the trustees deciced to cancel the 2020 PyCon UK event and there was no other significant activity during the course of the year.
```
Comment (trallard) [inline source/2021.tex:68]: Should this not be 2021?

**Comment by trallard** [comment]
LGTM - thank you all!

**Review by trallard — APPROVED** [review]

**Review by drvinceknight — APPROVED** [review]

**Comment by drvinceknight** [comment]
Merging as @doismellburning [did give a LGTM](https://github.com/UKPythonAssociation/trustees-annual-report/pull/2#issuecomment-981123086)

**Pull request merged**